### ISSUE TITLE
[FEATURE] Como un creador de concursos, quiero restringir los alias de identidades para evitar usuarios "apócrifos"

### ISSUE THREAD

**Issue by andreasantillana** [omegaup/omegaup#3405]
## Comportamiento Esperado

Antes de las identidades, la creación de usuarios en masa tenía restricción de superadmin o algún usuario específico dependiendo del tipo de alias del username. Por ejemplo, no cualquier usuario puede generar usuarios en masa del tipo `OMI20xx...` `OMIP20..`. 

## Comportamiento Actual

Cualquier usuario de omegaup puede crear usuarios nuevos usando los alias que normalmente usamos para las Olimpiadas.

## Posible Solución

Añadir un permiso ligado a un usuario para ciertos tipos de alias.

**Comment by heduenas** [comment]
Si alguien fuera ahorita y registrarla los alias de la OMI 2020 eso sería un problema para ustedes porque tendrían que usar un formato de alias diferente o por qué es un problema para ustedes esto?

**Comment by lhchavez** [comment]
> Si alguien fuera ahorita y registrara los alias de la OMI 2020 eso sería un problema para ustedes porque tendrían que usar un formato de alias diferente o por qué es un problema para ustedes esto?

sí, exactamente eso que dices. le restaría legitimidad al evento si los usuarios fueran algo así como "omi_official-2020:GUA01".

**Comment by andreasantillana** [comment]
Sí pasó. Ya no pudimos usar `OMI2020` para la OMI 2020. Lo bueno es que vamos a poder usar `OMI-2020`. Pero ojalá que sí se pueda restringir.
